Fix description about hot-region-cache-hits-threshold (#5721)
pingcap/docs: pd-control.md

    >> config set hot-region-schedule-limit 4       // 4 tasks of hot Region scheduling at the same time at most
    ```

> - `hot-region-cache-hits-threshold` is used to set the number of minutes required to identify the hot Region. Only when a Region is in the hot state for more than this number of minutes, it can participate in the hot scheduling.

Reviewer: ```suggestion
- `hot-region-cache-hits-threshold` is used to set the number of minutes required to identify a hot Region. PD can participate in the hotspot scheduling only after the Region is in the hotspot state for more than this number of minutes.
```
